oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -18,14 +18,11 @@
  */
 package oshi.software.os;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
 import org.junit.Test;
-
 import oshi.SystemInfo;
-import oshi.software.common.AbstractProcess;
+import oshi.software.os.windows.WindowsProcess;

```

Comment:
Yep, i totally missed that. I added a default AbstractProcess implementation for unit tests. That should do the trick.